.github/workflows/track-no-cover.yml

```diff
@@ -0,0 +1,40 @@
+---
+name: Check no cover Removal
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron:  "15 9 1 * *"
+
+jobs:
+  check-no-cover:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: "actions/setup-python@v2"
+        with:
+          python-version: "3.9"
+      - uses: extractions/setup-just@v1
+
+      - name: "Stop ignoring pragma: no cover lines"
+        run: |
+          set -eu
+          sed -i.bak 's/"pragma: no cover",/# "pragma: no cover",/' pyproject.toml
```

Comment:
Why is this in a separate step when we have multiple things going on in the other step?

Reply:
Vestige of a previous version that I meant to merge in, I'll pull it into the main step.

Reply:
Fixed and rebased.